definitions/control-v1.yaml

```diff
@@ -4582,6 +4591,21 @@ TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
           default: false
           description: If `true`, publishing messages with a push payload in the extras field is permitted and can trigger the delivery of a native push notification to registered devices for the channel.
           example: false
+        batchingEnabled:
+          type: boolean
+          default: false
+          description: If `true`, channels within this namespace will start batching inbound messages instead of sending them out immediately to subscribers.
+          example: false
+        batchingPolicy:
+          type: string
+          default: false
```

Comment:
`simple`

Reply:
The default batching policy will be the `simple` policy, which simply concatenates the incoming messages. Since default is set to `false` here I'm suggesting changing it to `simple` which is the string expected by the API

Reply:
Thanks :)

Reply:
see fixup 9cad89ad388eafec06bb086dc9638805ebb1ee1c